WalletWasabi/Gma/QrCodeNet/Encoding/DataEncodation/InputRecognition/ModeEncodeCheck.cs

```diff
@@ -25,10 +25,10 @@ private static bool EightBitByteCheck(string encodingName, string content)
 		/// Use given encoding to check input string from starting position. If encoding table is suitable solution.
 		/// it will return -1. Else it will return failed encoding position.
 		/// </summary>
-		/// <param name="content">input string</param>
-		/// <param name="encodingName">encoding name. Check ECI table</param>
-		/// <param name="startPos">starting position</param>
-		/// <returns>-1 if from starting position to end encoding success. Else return fail position</returns>
+		/// <param name="content">Input string</param>
+		/// <param name="encodingName">Encoding name. Check ECI table</param>
+		/// <param name="startPos">Starting position</param>
```

Comment:
Remove this line and rename the variable to `startingPosition` instead.

Reply:
Done.